Source: Elias Pipping <pipping@exherbo.org>
Upstream: http://lists.gnu.org/archive/html/bug-groff/2012-12/msg00014.html
Reason: Symlink to $(DESTDIR)
--- groff-1.22.1/contrib/mom/Makefile.sub
+++ groff-1.22.1/contrib/mom/Makefile.sub
@@ -160,7 +160,8 @@
 	done
 	for f in $(PDFDOCFILE); do \
 	  rm -f $(DESTDIR)$(pdfdocdir)/$$f; \
-	  ln -s $(DESTDIR)$(exampledir)/mom/$$f $(DESTDIR)$(pdfdocdir)/$$f; \
+	  mkdir -p $(DESTDIR)$(pdfdocdir); \
+	  ln -s $(exampledir)/mom/$$f $(DESTDIR)$(pdfdocdir)/$$f; \
 	done
 
 stamp-strip: $(STRIPFILES)
